tests/Fixer/Alias/ArrayPushFixerTest.php

```diff
@@ -113,6 +113,21 @@ public function provideFixCases(): iterable
             ',
         ];
 
+        yield 'simple traditional array' => [
+            '<?php $a[] = array($b, $c);',
+            '<?php array_push($a, array($b, $c));',
+        ];
+
+        yield 'simple short array' => [
+            '<?php $a[] = [$b];',
+            '<?php array_push($a, [$b]);',
+        ];
+
+        yield 'multiple element short array' => [
+            '<?php $a[] = [[], [], $b, $c];',
+            '<?php array_push($a, [[], [], $b, $c]);',
+        ];
+
```

Comment:
can you add:
``php
        yield 'second argument wrapped in `(` `)`' => [
            '<?php $a::$c[] = ($b);',
            '<?php array_push($a::$c, ($b));',
        ];
```